src/utils/positionUtils.ts

```diff
@@ -254,30 +254,38 @@ export const calcTradeFee = ({
   isMaker: boolean
   direction: PositionSide
 }) => {
-  const noValue = { total: 0n, impactFee: 0n, skewFee: 0n, feeBasisPoints: 0n }
+  const noValue = { total: 0n, feeBasisPoints: 0n, proportionalFee: 0n, linearFee: 0n, adiabaticFee: 0n }
```

Comment:
I'm wondering if it makes sense to have this be the `returnValue` and then each second modifies the fields, this would ensure consistency of types. Alternatively we just add an explicit type annotation to this function. Sometimes the types can get out of sync for early returns which can be annoying